examples/12_Predicting_Ki_of_Ligands_to_a_Protein.ipynb

```diff
@@ -0,0 +1,1286 @@
+            "text/plain": [
+              "<IPython.core.display.Image object>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "image/png": "iVBORw0KGgoAAAANSUhEUgAAASwAAAEsCAIAAAD2HxkiAAAABmJLR0QA/wD/AP+gvaeTAAAgAElEQVR4nO3deVxU5f4H8M+wL6MCkkuGIC6omJqoaWgpcEVvpHaTvC1kXX9StqDeW2JhUZaG5sLFTLEyuV1bcAfNhaupuFSi5QKoyKKouLDJIsM2z++PhxAVmYE55zwzw/f98uULxjPP+YLzmeeZ55zzHBVjDIQQcSxEF0BIa0chJEQwCiEhglEICRGMQkiIYBRCQgSjEBIiGIWQEMEohIQIRiEkRDAKISGCUQgJEYxCSIhgFEJCBKMQEiIYhZAQwSiEhAhGISREMAohIYJRCAkRjEJIiGAUQkIEoxASIhiFkBDBKISECEYhJEQwCiEhglEICRGMQkiIYBRCQgSjEBIiGIWQEMEohIQIRiEkRDAKISGCUQgJEczYQ7h2LVQqvP32HQ+mpEClwtixgmoiRFLGHkJCzB6FkBDBKISECEYhJEQw0wjhkiVQqW7/GTJEdEGESMdKdAF6GTAAw4bd/vbGDWzaJK4aQiRlGiEMCMDixbe/TUmhEBLzYRrDUULMGIWQEMEohIQIZiYhXLMG/fvD0RFubnjtNRQViS6IEL2ZxsRM0777DqGhWLkSY8ciKwsvv4z8fGzYILosQvSjYoyJrsFQH3yA8nIsWVL37eLFmDcPJSVCayJEb+bQE86bd8e3WVno0kVQKYQ0nzmEsKENG7B6NbZtE10HIXozk4kZLiYGL76I1avpUkNiSky7JzxwAD4+cHREbS2mT8eWLfjpJ/j5iS6LkOYw7YkZV1fs24c+ffD88zh5Ejt2wMNDdE2ENJNp94RcVBR27MChQ3ByQnFx3YNt28LCrMbaxGyZQwhjYlBaiv7973jw1Cn06yeoIEKaw+g6C76yk40NLl68/WBNze2VnS5ehJNT3Z/CQjz2GCor0a4d1q0DY7f/UAKJqTDSnrC6GgsXYsWKRv7pwQfxxx91Xw8ahB9+QK9eAODqqlx5hEjI6HpCzssLa9bg6tVG/snKCh4edX8sLPDgg3Vfq9WKV0mIFIw0hKGhqKy840JeQsyVkYbQwwMTJ2LVKhQUNLVZfj599iMmz0hDqNUiIgLl5YiOFl0KITIz0hAC8PFBYCCWL8fNm6JLIUROxhtCABERuHmz8TlSQsyGUYdw5EiMHIlly3Drll7bf/klIiPpSkJiYow6hAAiIpCfj6++grW1ji01Grz/PubNg6cnFi9GRYUi9RFiMGMPYWAgBg/G4sVo107HlnZ22LIFfn4oKMA778DDAwsXorJSkSoJMYCxhxDAe+8hLw/5+bq3HDYMe/YgKQmDB+P6dcyZAy8vrF6N2lr5qySkpYwohEVFSE9v5PGJE+Ht3Yx2AgLw229ISED//rhwAa++iv79sX49TPmaLWLOjOV6woICjBmDS5fqrg+UhFaLjRvx7rvIzASARx/F/Pnw95emcUKkYhQ94Y0b8PPD8eNwckKbNpI1a2GB4GCkpyM2Fp0749dfERCAESOQnCzZLggxnPie8No1/OUvOHUKvXtjzx48+KAseykvx+efY+HCunWBAwLw2WcYOFCWfRHSLIJDePUq/P2RloY+fbBnDzp3lnd3paX44gssWICSElhY4JlnsGABevSQd6eENE1kCHNz4eeH8+cxcCCSkpS7IDA/H4sX49//hkYDa2u88g
```

Comment:
As one suggestion, we should consider renaming `IndiceSplitter` to `SpecifiedSplitter`. That could reduce our breakage on the tutorials.

On a separate note, we should go back and add explicit failures if people try to import the splitters we removed (raising a `ValueError` and saying they should use X splitter instead). We can do this in subsequent PRs though

Reply:
> On a separate note, we should go back and add explicit failures if people try to import the splitters we removed (raising a ValueError and saying they should use X splitter instead). We can do this in subsequent PRs though

Yes! I will make the PR.
And, I revert tutorial 12 because of avoiding conflicts.